stage1/init/init.go

```diff
@@ -624,8 +625,8 @@ func stage1() int {
 	}
 
 	// kvm flavor has a bit different logic in handling pid vs ppid, for details look into #2389
-	// it does not require existance of "ppid", but instead registers current pid (which
-	// will be reused by lkvm binary) as an pod process pid used in entering
+	// it doesn't require the existence of a "ppid", instead it registers the current pid (which
```

Comment:
FYI, some comments were superfluous, like "prepare mounts for kvm flavor" and the code reads as 
```go
if flavor == "kvm" {
    KvmPrepareMounts................
    ...
```